service/src/main/kotlin/io/provenance/explorer/grpc/v1/MarkerGrpcClient.kt

```diff
@@ -57,5 +57,16 @@ class MarkerGrpcClient(channelUri: URI) {
             QueryHoldingResponse.getDefaultInstance()
         }
 
+    fun getAllMarkerHolders(denom: String): QueryHoldingResponse =
+        try {
+            markerClient.holding(
```

Comment:
This will not get the whole holder list. I would use the existing function (right above) and cycle through the pages. That way youll get all holders.

Reply:
And if thats too much in-memory data, you might want to start saving the nhash holders in a table with current balances.

Reply:
Saving paginated data to a table.